cmd/api/main.go

```diff
@@ -93,10 +93,14 @@ func main() {
 		log.Fatal().Err(err).Msg("creating parser")
 	}
 
+	// Shared memory
```

Comment:
Creates the shared memory object that will be shared with [`EventFeed`](https://github.com/tablelandnetwork/go-tableland/pull/508/files#diff-ae22ad1e20b89eaefa8e09e4ed89443ac10fdc555b9767625f3866e3853100afR247) and [Gateway's resolver](https://github.com/tablelandnetwork/go-tableland/pull/508/files#diff-ae22ad1e20b89eaefa8e09e4ed89443ac10fdc555b9767625f3866e3853100afR473)